### PR TITLE
Cleaned up effect.dm

### PR DESCRIPTION
**Pull request by despotate**
effect.dm is now standardized to have two newlines between each effect and one newline between each effect's methods. Nothing special about these quantities, but the file was previously a complete unstandardized fucking mess where you'd see a several-method pileup in one part, a mexican standoff of effects in another, and misplaced effects with the wrong stage everywhere else, which are now in their correct spots. A couple of full-path issues were also resolved.

In summary:

- Effects with stages that didn't match the section they were in are now in their proper sections.
- Space-adaptation moved to special effects (someone should update the wiki, it hasn't been stage 3 in a long time).
- Spaces standardized.
- Full paths added.

This sets the ground for a deeper rework of viral effects, which is what I'm planning on doing later.

### CODE REVIEW

File: code/modules/virus2/effect.dm [hunk 1]
```diff
@@ -76,24 +79,42 @@
...
 
-/datum/disease2/effect/invisible
-	name = "Waiting Syndrome"
-	stage = 1
-	activate(var/mob/living/carbon/mob,var/multiplier)
-		return
 
+/datum/disease2/effect/spaceadapt
+	name = "Space Adaptation Effect"
+	stage = 5
```
Comment (Probe1) [inline code/modules/virus2/effect.dm:88]: I've been too busy to look but I've suspected this was broken.  And now I know.

WHOOOOOOOOOO
EDIT: IT WAS POMF IN A DIRECT COMMIT TO THE REPO
